force_torque_sensor_broadcaster/src/force_torque_sensor_broadcaster.cpp

```diff
@@ -87,12 +90,39 @@ controller_interface::CallbackReturn ForceTorqueSensorBroadcaster::on_configure(
         torque_names.z));
   }
 
+  try
+  {
+    filter_chain_ =
+      std::make_unique<filters::FilterChain<WrenchMsgType>>("geometry_msgs::msg::WrenchStamped");
+  }
+  catch (const std::exception & e)
+  {
+    fprintf(stderr, "Exception thrown during filter chain creation with message : %s \n", e.what());
+    return CallbackReturn::ERROR;
+  }
+
+  // As the sensor_filter_chain parameter is of type 'none', we cannot directly check if it is
+  // present. Even if the sensor_filter_chain parameter is not specified, the filter chain will be
+  // correctly configured with an empty list of filters (https://github.com/ros/filters/issues/89).
```

Comment:
how to proceed here?

Reply:
filters 2.2.2 is released, and only the sync to jazzy is missing. So we can update this PR already, and just wait for the backport to jazzy until it got synced.

Reply:
Shouldn't we use `get_length()` now?